lib/.gitignore

```diff
@@ -36,3 +36,4 @@ mdfs*.log
 /VixDiskLib/lib
 /VixDiskLib/vddklib
 
+vendor/
```

Comment:
I will remove it into its own PR unless I discover I messed up and it is really not needed.